parser-typechecker/src/Unison/Codebase/Editor/HandleInput.hs

```diff
@@ -1291,6 +1295,92 @@ loop = do
           BranchUtil.makeDeleteTermName (resolveSplit' (HQ'.toName <$> hq))
         go r = stepManyAt . fmap makeDelete . toList . Set.delete r $ conflicted
 
+      ReplaceI from to patchPath -> do
```

Comment:
This duplicates quite a lot of code that's already duplicated twice in this file. It would be best to move that code into a function that gets called from here with different arguments depending on whether the user typed `replace.term`, `replace.type`, or just `replace`.

I feel (not very strongly) that `replace.term` and `replace.type` should just go away.